src/sage/graphs/graph.py

```diff
@@ -4932,6 +4939,9 @@ def minor(self, H, solver=None, verbose=0, *, integrality_tolerance=1e-3):
           solvers over an inexact base ring; see
           :meth:`MixedIntegerLinearProgram.get_values`.
 
+        - ``induced`` -- boolean (default: ``False``); if ``True``, returns an
+            induced minor isomorphic to `H` if it exists, and ``None`` otherwise.
```

Comment:
When it is not possible to find a minor, the method raises an error. So it will never return ``None``.